index.html

```diff
@@ -208,5 +208,31 @@ <h2>Skills</h2>
         </ul>
     </section>
   </main>
+  <footer id="footer-contact">
+    <section id="contact-form">
+      <p id="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <form action="https://formspree.io/f/mnqreoov" method="post">
+        <div id="input-name" class="input-wrapper">
+          <input type="text" required  maxlength="30" placeholder="Full Name">
+        </div>
+        <div id="input-email" class="input-wrapper">
+          <input type="email" name="email" required placeholder="Email Adress">
+        </div>
+        <textarea required name="message" id="" cols="30" rows="10" maxlength="500" placeholder="Enter text here"></textarea>
+        <center><button class="btn" type="submit" id="action">Get in touch</button></center>
+      </form>
+    </section>
+    <section id="social-links">
+      <div class="social-icons">
+        <img src="img/Github.svg" alt="">
+        <img src="img/LinkedIn.svg" alt="">
+        <img src="img/Hand.svg" alt="">
+        <img src="img/Twitter.svg" alt="">
+        <img src="img/M.svg" alt="">
```

Comment:
- [ ] The required alt attribute specifies an alternate text for an image if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of a slow connection, an error in the src attribute, or if the user uses a screen reader) ☺️ . I suggest that you provide a descriptive `alt text` 👍 .

Reply:
I have done as I am advised